mathics/doc/online.py

```diff
@@ -20,8 +20,11 @@ def online_doc_string(
 
     # First look at user definitions:
     for rulemsg in ruleusage:
-        if rulemsg.pattern.expr.elements[1].__str__() == '"usage"':
-            usagetext = rulemsg.replace.value
+        if rulemsg.pattern.expr.get_elements()[1].__str__() == '"usage"':
+            # mypy complains about the plane conversion, but
+            # to_python() returns a string wrapped in quotes.
+            # So, let's get ride the quotes...
+            usagetext = rulemsg.get_replace_value().to_python()[1:-1]
```

Comment:
```suggestion
            usagetext = rulemsg.get_replace_value().to_python(use_quotes=False)
```

Reply:
I don't understand the comment regarding mypy. Please explain. The previous pytest CI failure was about the test not expecting quotes around the usage string. 

What exactly is mypy "complaining" about?

Reply:
It just means that I cannot just write `rulemsg.get_replace_value().value`, because `get_replace_value` returns a `BaseElement`, not a `String`.

Reply:
Ok. Using `use_quotes` is clearer, and more reliable than `[1:-1]`.